packages/app/obojobo-repository/shared/components/notification.jsx

```diff
@@ -0,0 +1,68 @@
+const React = require('react')
+require('./notification.scss')
+
+const Notification = ({ onDataFromNotification }) => {
+	const [notifications, setNotifications] = React.useState([])
+
+	React.useEffect(() => {
+		if (document && document.cookie) {
+			const cookiePropsRaw = decodeURIComponent(document.cookie).split(';')
+
+			let parsedValue
+			cookiePropsRaw.forEach(c => {
+				const parts = c.trim().split('=')
+				if (parts[0] === 'notifications') {
+					parsedValue = JSON.parse(parts[1])
+				}
+			})
+
+			const parsedNotifications = parsedValue
+			setNotifications(parsedNotifications)
+		} else {
+			// there is nothing to render
+		}
+	}, [])
+
+	function onClickExitNotification(key) {
+		onDataFromNotification(notifications.length - 1)
+		setNotifications(prevNotifications => prevNotifications.filter((_, index) => index !== key))
+	}
+
+	React.useEffect(() => {
+		const jsonNotifications = JSON.stringify(notifications)
+		const cookieString = `${encodeURIComponent(jsonNotifications)};`
+		document.cookie = 'notifications=' + cookieString
+	}, [notifications])
+
+	const renderNotification = (key, text, title) => {
+		return (
+			<div className={`notification-banner`} key={key}>
+				<div className="notification-header">
+					<h1 tabindex="0">{title}</h1>
```

Comment:
It seems to be functionally fine, but it's worth noting that when using React 'tabindex' properties need to be written in camel-case as 'tabIndex' to be considered valid.